src/action/sample.js

```diff
@@ -0,0 +1,10 @@
+// Moduler Importations
+import * as types from "../actionTypes/sample";
```

Comment:
#### Critical
Avoid wildcard imports as much as possible. You can check [here](https://github.com/airbnb/javascript#modules) to get more context

Reply:
Alright, I will implement the feedback